src/serializer/index.ts

```diff
@@ -23,22 +27,51 @@ export class Serializer<
 			: R
 		: FirstOut,
 > {
+	private queue = Symbol('serializerQueue');
```

Comment:
Symbol is used so an unique queue identifier is made for each Serializer instance